ballerina-shell/modules/shell-core/src/test/java/io/ballerina/shell/test/evaluator/ActionEvaluatorTest.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) 2023, WSO2 LLC. (http://wso2.com) All Rights Reserved.
```

Comment:
```suggestion
 * Copyright (c) 2024, WSO2 LLC. (http://wso2.com) All Rights Reserved.
```

Reply:
Need to remove the "All Rights Reserved" as well.